department-of-veterans-affairs/vets-website
# 74348c Add Continuation of Medicare status page

## Summary

This PR introduces a custom page component for use in form 10-10D. A custom page component was created because this section of the form needed dynamic wording functionality. This component will be added to form 10-10D's Applicant Information section in an upcoming PR. For now, it is just being added without being integrated so that the PR stays within a reasonable size. Additional unit testing is planned in a followup PR.

- Added a custom page for collecting which parts of medicare a user is enrolled in
- Added corresponding custom review page component
- Team: IVC-CHAMPVA
- Flipper: NA, form is not set to go to prod

## Related issue(s)

- https://github.com/department-of-veterans-affairs/va.gov-team/issues/74348

## Testing done

- Manual testing to verify fields behave as expected
- Ran yarn test:coverage-app ivc-champva/10-10D to verify existing tests run and pass

## Screenshots
|         | Scenario 1 | Scenario 2 |
| ------- | ------ | ----- |
| Desktop | ![medicare_con1](https://github.com/department-of-veterans-affairs/vets-website/assets/18408628/faccce1f-5117-42f0-acf3-e54b1880ae36) |![medicare_con](https://github.com/department-of-veterans-affairs/vets-website/assets/18408628/a80cf9fc-114a-4794-8f79-ab1aab1e1108) |

## What areas of the site does it impact?

This form only

## Acceptance criteria

### Quality Assurance & Testing


### Error Handling


### Authentication


### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:


## Requested Feedback

NA
